website/docs/r/emr_cluster.html.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_emr_cluster"
+sidebar_current: "docs-alicloud-resource-emr-cluster"
+description: |-
+  Provides a EMR Cluster resource.
+---
+
+# alicloud\_emr\_cluster
+Provides a EMR Cluster resource. With this you can create, read, and release  EMR Cluster. 
+
+-> **NOTE:** Available in 1.56.0+.
+
+## Example Usage
+```
+resource "alicloud_emr_cluster" "default" {
+    name = "terraform-test"
+
+    emr_ver = "EMR-3.22.0"
+
+    cluster_type = "HADOOP"
+
+    host_group {
+        host_group_type = "MASTER"
+        node_count = "2"
+        instance_type = "ecs.g5.xlarge"
+        disk_type = "CLOUD_SSD"
+        disk_capacity = "80"
+        disk_count = "1"
+        sys_disk_type = "CLOUD_SSD"
+        sys_disk_capacity = "80"
+    }
+
+    host_group {
+        host_group_type = "CORE"
+        node_count = "3"
+        instance_type = "ecs.g5.xlarge"
+        disk_type = "CLOUD_SSD"
+        disk_capacity = "80"
+        disk_count = "4"
+        sys_disk_type = "CLOUD_SSD"
+        sys_disk_capacity = "80"
+    }
+
+    high_availability_enable = true
+    option_software_list = ["HBASE","PRESTO",]
+    zone_id = "cn-huhehaote-a"
+    security_group_id = "${security_group_id}"
+    is_open_public_ip = true
+    charge_type = "PostPaid"
+    vpc_id = "${vpc_id}"
+    vswitch_id = "${vswitch_id}"
+    net_type = "vpc"
+    user_defined_emr_ecs_role = "${user_defined_emr_ecs_role}"
+    ssh_enable = true
+    master_pwd = "ABCtest1234!"
+}
+
+```
+
+## Argument Reference
+The following arguments are supported:
+
+* `name` - (Required) The name of emr cluster.
+* `emr_ver` - (Required, ForceNew) EMR Version, e.g. EMR-3.22.0. 
```

Comment:
Please provide all of valid versions.

Reply:
> Please provide all of valid versions.

The versions will change(old version off-line and new version on-line) along with very release of EMR. User can fetch the valid value from web console.